src/core/financial.jl

```diff
@@ -134,8 +208,245 @@ struct Financial
             microgrid_upgrade_cost_pct,
             macrs_five_year,
             macrs_seven_year,
+            CO2_cost_per_tonne,
+            CO2_cost_escalation_pct,
+            NOx_grid_cost_per_tonne,
+            SO2_grid_cost_per_tonne,
+            PM25_grid_cost_per_tonne,
+            NOx_onsite_fuelburn_cost_per_tonne,
+            SO2_onsite_fuelburn_cost_per_tonne,
+            PM25_onsite_fuelburn_cost_per_tonne,
+            NOx_cost_escalation_pct,
+            SO2_cost_escalation_pct,
+            PM25_cost_escalation_pct,
             offgrid_other_capital_costs,
             offgrid_other_annual_costs
         )
     end
-end
\ No newline at end of file
+end
+
+
+# Convert from 2010$ to 2020$ (source: https://www.in2013dollars.com/us/inflation/2010?amount=100)
+const USD_2010_to_2020 = 1.246
```

Comment:
This value is only used in `easiur_costs`. Why does it need to be a `const`?

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/28/commits/c61f79a4263245f6662a777f05362186c258450d